extensions/azure-functions-http/runtime/src/main/java/io/quarkus/azure/functions/resteasy/runtime/BaseFunction.java

```diff
@@ -34,13 +34,19 @@
 public class BaseFunction {
     private static final Logger log = Logger.getLogger("io.quarkus.azure");
 
-    protected static String deploymentStatus;
-    protected static boolean started = false;
-    protected static boolean bootstrapError = false;
+    protected static volatile String deploymentStatus;
```

Comment:
They don't need to be volatile if everything is done in a static block

Reply:
I've removed them.